ni_measurementlink_service/session_management/_types.py

```diff
@@ -305,18 +305,32 @@ class SessionInitializationBehavior(IntEnum):
 
     INITIALIZE_SERVER_SESSION = 1
     """
-    Require the NI gRPC Device Server to initialize a new session with the
-    specified name.
+    Initialize a new session with the specified name.
 
     Note: When using the Session as a context manager and the context exits, it
     will automatically close the server session.
     """
 
     ATTACH_TO_SERVER_SESSION = 2
     """
-    Require the NI gRPC Device Server to attach to an existing session with the
-    specified name.
+    Attach to an existing session with the specified name.
 
     Note: When using the Session as a context manager and the context exits, it
     will detach from the server session and leave it open.
     """
+
+    INITIALIZE_SESSION_THEN_DETACH = 3
+    """
+    Initialize a new session with the specified name.
+
+    Note: When using the Session as a context manager and the context exits, it
```

Comment:
For the two new enum values, please highlight the asymmetry instead of hiding it.

```
"""
Initialize a new session. When exiting the context manager, detach instead of closing.

Note: This initialization behavior is intended for TestStand code modules used in ProcessSetup.
"""
```

Likewise for the other one.